### PR TITLE
Fix `.form-control-plaintext` for select, textarea

### PR DESCRIPTION
**Pull request by WinterSilence**
hide `select` "arrow" (icon at right center) and `textarea` "resizer" (icon at right bottom)

### CODE REVIEW

**Review by ffoodd — CHANGES_REQUESTED** [review]
Thanks!

I'm not completely sure about the need for this, but I don't see any reason not to include this.

BTW would you mind adding `selet` and `textarea` examples to the [Readonly plain text docs part](https://getbootstrap.com/docs/5.1/forms/form-control/#readonly-plain-text)?

File: scss/forms/_form-control.scss [hunk 1]
```diff
@@ -164,6 +165,10 @@ textarea {
   &.form-control-lg {
     min-height: $input-height-lg;
   }
+
+  &.form-control-plaintext {
+    resize: none;
+  }
```
[the remark this answers is not in the document]
Comment (WinterSilence) [reply scss/forms/_form-control.scss:171]: @ffoodd it's fix for `textarea` - by default, only textareas can resize, i'm don't tested how it with `select`/`input`

**Comment by WinterSilence** [comment]
@ffoodd 
> BTW would you mind adding selet and textarea examples to the [Readonly plain text docs part](https://getbootstrap.com/docs/5.1/forms/form-control/#readonly-plain-text)?

ok!

File: scss/forms/_form-control.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (ffoodd) [reply scss/forms/_form-control.scss:171]: I suppose it'd do nothing: if you add a working example for `select` and `texteea` to the docs, you'll be able to try.

I think it's safe to add this to the class directly.

[the remark this answers is not in the document]
Comment (WinterSilence) [reply scss/forms/_form-control.scss:171]: @ffoodd some browsers generate notice/warning if rule in wrong context.
> Applies to	elements with [overflow](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow) other than visible, and optionally replaced elements representing images or videos, and iframes

I'm can't test how it work in Safari

File: site/content/docs/5.1/forms/form-control.md [hunk 2]
```diff
@@ -57,6 +57,22 @@ If you want to have `<input readonly>` elements in your form styled as plain tex
...
+    </div>
+  </div>
+  <div class="mb-3 row">
+    <label for="staticGender" class="col-sm-2 col-form-label">Gender</label>
+    <div class="col-sm-10">
+      <select readonly class="form-control-plaintext" id="staticGender">
+        <option value="0" selected>male</option>
+        <option value="1">female</option>
+        <option value="2">other</option>
+      </select>
```
Comment (ffoodd) [inline site/content/docs/5.1/forms/form-control.md:73]: I suggest using numbers for options here, just like in the select docs page.

File: scss/forms/_form-control.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (ffoodd) [reply scss/forms/_form-control.scss:171]: I'll try to make a Codepen to try this. Not sure when, though: please be patient 😄

**Pull request closed without merging**